crates/store/re_datafusion/src/dataframe_query_common.rs

```diff
@@ -190,6 +191,49 @@ impl DataframeQueryTableProvider {
             chunk_request,
         })
     }
+
+    fn column_to_selector(column: &Column) -> Option<ComponentColumnSelector> {
+        ComponentColumnSelector::from_str(column.name()).ok()
+    }
+
+    fn compute_column_is_not_null_filter(
```

Comment:
Maybe this would read slightly better?

```suggestion
    fn compute_column_is_neq_null_filter(
```